modules/ui.styl

```diff
@@ -200,10 +201,18 @@ formElementBase() // This could be a class that is extended rather than a mixin
       color: var(--color-black-soft)
       cursor: not-allowed
 
+  &--small-text
```

Comment:
Not sure what style in the Figma designs with would relate to. Important that this library is the single source of code truth for visual styles, but that all the styles built in the library relate to exact designs on Figma. If there were things on the donor site that aren't replicated in the library that means either Pietro & co need to design solutions or the donor site needs to adopt a new solution that is already found in the library.

Reply:
For example there the font size is unlikely to be correct as doesnt relate to any of the specific font sizing scale built out elsewhere in the library

Reply:
Found the small variation of this